### ISSUE TITLE
bakta input directory error

### ISSUE THREAD

**Issue by Kvass-Toto** [bactopia/bactopia#272]
**Description**
While using the same directory syntax for different tools, bakta throws an error saying it cannot find the post-assembly inputs. Other tools don't throw an error for the exact same directory and everything runs fine.

**Steps to Reproduce**
```
bactopia --wf kleborate --bactopia ./data_1

(everything runs great)

bactopia --wf bakta --bactopia ./data_1

--------------------------------------------------------------------
Required parameters are missing, please check and try again.


Required Parameters
  --bactopia                          [string]  The path to bactopia results to use as inputs

--------------------------------------------------------------------
ERROR: Validation of pipeline parameters failed!
Please correct to continue
```

**Expected Behavior**
Inside the ./data_1 folder are the output folders (Sample_1, Sample_2, ect.) containing kmers, assemblies, QC, ect. Tools like pangenome and kleborate run great. I double checked to make sure that I ran all the tools in the same directory, pointing to ./data_1 (one level below). 

**Execution Environment**

- Bactopia Version: bactopia_version2
- OS: Mac
- Environment: Conda

**Additional Information**
Thank you so much!

**Comment by rpetit3** [comment]
Definitely a bug, it's not reporting `--bakta_db` as missing.  Try using `--bakta_db` which should point to a Bakta db.

I'll get the usage fixed in the meantime

**Comment by Kvass-Toto** [comment]
I tried this:

bactopia --wf bakta_db --bactopia ./data_1

and got:

bakta_db is not an available workflow or Bactopia Tool.

**Comment by rpetit3** [comment]
Try this: `bactopia --wf bakta --bactopia ./data_1 --bakta_db <PATH TO BAKTADB>`

**Comment by rpetit3** [comment]
Hi @Kvass-Toto, 

Did the above comment help you out?

**Comment by rpetit3** [comment]
I've released v2.0.2 (https://github.com/bactopia/bactopia/releases/tag/v2.0.2) which includes a fix for this issue, but also many other changes! I'm going to go ahead and close this for now, please feel free to reopen!

If you have a moment, could you take a few minutes to fill out a [Bactopia Usage Survey](https://forms.gle/PTgUxEBwCsCdbQjr7)? It really helps me direct the direction of Bactopia's development! Thanks!

**Comment by lifenske** [comment]
Hi, I'm trying to use bakta but it ended up in the message:

`WARN: Access to undefined parameter 
backta_db -- Initialise it to a default value eg. params.backta_db = some_value`

`Cannot invoke method startsWith() on null object`

`backta_db` also looks like a typo, not sure what went wrong there.

My command was:

`bactopia --wf bakta --bactopia agalactiae/ --bakta_db datasets/bakta/db/` 

Thank you for helping.
